### PR TITLE
Add autochecker

### PR DESCRIPTION
**Pull request by Jachokoreto**

### CODE REVIEW

File: checker.js [hunk 1]
```diff
@@ -1,31 +1,72 @@
 const fs = require("fs");
-const core = require("@actions/core");
```
Comment (easonchai) [inline checker.js:2]: We should use the core package for the conditional action run handling

File: checker.js [hunk 2]
```diff
@@ -1,31 +1,72 @@
...
+        console.log(parsed[i], p_puzzle[i], i);
+        return false;
+      }
+    }
+  }
   return true;
 }
 
-function validAnswer(parsed) {
+function validAnswer(parsed, p_puzzle) {
```
Comment (easonchai) [inline checker.js:20]: We can remove `p_puzzle` from here since it is unused

File: checker.js [hunk 3]
```diff
@@ -1,31 +1,72 @@
...
 function readFile() {
   try {
     const answer = fs.readFileSync("./solution.txt", "utf8");
-    checkAnswer(answer);
+    var puzzle = fs.readFileSync("./README.md", "utf8");
+    puzzle = puzzle.slice(puzzle.length -256, puzzle.length -3);
+    checkAnswer(answer, puzzle);
   } catch (err) {
-    core.error(err);
+    console.error(err);
```
Comment (easonchai) [inline checker.js:69]: All `console.error` can be swapped with `core.error`

**Review by easonchai — CHANGES_REQUESTED** [review]

File: checker.js (hunk 1 above)
Comment (Jachokoreto) [reply checker.js:2]: Added.

File: checker.js (hunk 2 above)
Comment (Jachokoreto) [reply checker.js:20]: Removed

File: checker.js (hunk 3 above)
Comment (Jachokoreto) [reply checker.js:69]: Swapped all.

Comment (Jachokoreto) [reply checker.js]: Oops i changed the console.log too lol

**Review by easonchai — APPROVED** [review]

**Pull request merged**